### ISSUE TITLE
Change Logo in README file

### ISSUE THREAD

**Issue by arshadkazmi42** [arshadkazmi42/general-feedback#13]
**Describe the bug**
Change logo in readme file

**To Reproduce**
NA

**Expected behavior**
NA

**Screenshots**
NA

**Desktop (please complete the following information):**
 - Brand: [e.g. Moto G2] NA
 - Version [e.g. Nougat] NA

**Additional context**
Change the logo present at the top of the README file with this logo

![gf_icon_200x200px](https://user-images.githubusercontent.com/4654382/45953946-3b8ba600-c029-11e8-8723-4c94e9fa3a56.png)

Basically this image needs to be added in assets folder, with name logo.png, and existing logo.png image needs to be removed.

**Comment by arshadkazmi42** [comment]
Thanks @wilsantow for working on this.
Closing issue as this has been fixed and merged to master here #15
